packages/pluggableWidgets/accordion-web/src/Accordion.tsx

```diff
@@ -5,19 +5,42 @@ import { Accordion as AccordionComponent, AccordionGroups } from "./components/A
 import { useIconGenerator } from "./utils/iconGenerator";
 
 import { AccordionContainerProps } from "../typings/AccordionProps";
+import { Header } from "./components/Header";
 
 export function Accordion(props: AccordionContainerProps): ReactElement | null {
     const accordionGroups: AccordionGroups | undefined = useMemo(() => {
-        if (props.groups.some(group => group.visible.value === undefined || group.headerText.value === undefined)) {
+        if (
+            props.groups.some(
+                group =>
+                    group.visible.value === undefined ||
+                    group.headerText.value === undefined ||
+                    group.initiallyCollapsed.value === undefined ||
+                    (group.collapsed && group.collapsed.value === undefined)
```

Comment:
Why is `collapsed` the only function that needs a non-nullable check? 🤔

Reply:
Because it's the only optional prop of the four seen in the code snippet. Changing line 18 to `group.collapsed?.value === undefined` would break the behaviour, since in the case of a nullable value the outcome of the expression will be true, which is something we don't want here.

Reply:
Clear!